### PR TITLE
Add device to support Trust Remote control ZYCT-202

### PR DESCRIPTION
**Pull request by hwijers**
https://github.com/Koenkk/zigbee2mqtt/issues/635

### CODE REVIEW

File: converters/fromZigbee.js [hunk 1]
```diff
@@ -1773,7 +1773,39 @@ const converters = {
             return {action: `${direction}`};
         },
     },
-
+    ZYCT202_on: {
+        cid: 'genOnOff',
+        type: 'cmdOn',
+        convert: (model, msg, publish, options) => {
+            return {action: msg.groupid+'_on'};
```
Comment (Koenkk) [inline converters/fromZigbee.js]: I think it's nicer to have `action: 'on', groupid: msg.groupid` here, perhaps some people don't want to act on the group id, for that case they now have to respond to 5 different actions. I don't see a downside in separating them either.

Comment (hwijers) [reply converters/fromZigbee.js]: How can I do in Home Assistant in an automation a state check at state 'on' and groupid '45755' then switch 'Y'?
You now assume that people who have this remote control do not use the 1,2,3,4,5 and Group selector?

Comment (Koenkk) [reply converters/fromZigbee.js]: You can do:

```yaml
automation:
  - alias: Respond to button clicks
    trigger:
      platform: mqtt
      topic: 'zigbee2mqtt/<FRIENDLY_NAME'
    condition:
      - condition: template
        value_template: '{{ "on" == trigger.payload_json.action }}'
      - condition: template
        value_template: '{{ 1234 == trigger.payload_json.groupid }}'
    action:
      entity_id: light.bedroom
      service: light.toggle
```

Comment (hwijers) [reply converters/fromZigbee.js]: Ok. I have to learn to let go the automation rules based on entity and state, and go to work with mqtt topics and conditions.
I changed it...

**Comment by Koenkk** [comment]
Thanks!

**Comment by hwijers** [comment]
Thank you!

**Pull request merged**